proto/common/common.proto

```diff
@@ -67,11 +67,16 @@ message Eos {
 message TapEvent {
   TcpAddress source = 1;
   TcpAddress target = 2;
+  EndpointMeta target_meta = 4;
 
   oneof event {
     Http http = 3;
   }
 
+  message EndpointMeta {
+    map<string, string> labels = 1;
```

Comment:
Related to question above, should we consider a more generalized/shared `Labels` or `LabelSelector` type?

Reply:
we could have a common Labels type that just has a `map<string, string>` -- i don't think it buys us that much